tests/testdata/codegen/dashed-import-schema/docs/tree/v1/nursery/_index.md

```diff
@@ -15,7 +15,111 @@ no_edit_this_page: true
 
 
 
-## Create Nursery Resource {#create}
+## Creating a Nursery Resource
+<div>
+<pulumi-chooser type="language" options="typescript,python,go,csharp,java,yaml"></pulumi-chooser>
+</div>
+
+
+<div>
+    <pulumi-choosable type="language" values="csharp">
+        <div class="highlight">
+            <pre class="chroma">
+                <code class="language-csharp" data-lang="csharp">
+var nurseryResource = new Plant.Tree.V1.Nursery("nurseryResource", new()
+{
+    Varieties = new[]
+    {
+        Plant.Tree.V1.RubberTreeVariety.Burgundy,
+    },
+    Sizes = 
+    {
+        { "string", Plant.Tree.V1.TreeSize.Small },
+    },
+});
+                </code>
+            </pre>
+        </div>
+    </pulumi-choosable>
+</div>
+
+<div>
+    <pulumi-choosable type="language" values="go">
+        <div class="highlight">
+            <pre class="chroma">
+                <code class="language-go" data-lang="go">
+example, err := tree.NewNursery(ctx, "nurseryResource", &tree.NurseryArgs{
+Varieties: treev1.RubberTreeVarietyArray{
+tree.RubberTreeVarietyBurgundy,
+},
+Sizes: treev1.TreeSizeMap{
+"string": tree.TreeSizeSmall,
+},
+})
```

Comment:
Do we need to format the generated go code so it's indented properly?

Reply:
We actually do try and format the Go code after generating the program. However if the program has a syntax error due to a `%PANIC` present, then formatting fails and we fallback to the non-formatted code which is why you get this non-indented version.